packages/lab/src/input-next/InputNext.css

```diff
@@ -189,3 +186,22 @@
   padding-left: var(--salt-size-adornmentGap);
   column-gap: var(--salt-size-adornmentGap);
 }
+
```

Comment:
this is a hack for changing border size without the height changing - otherwise we will need an Activation Indicator div